### PR TITLE
feat: add rules for trailing commas in function calls and closure `use`

### PR DESCRIPTION
**Pull request by simPod**
Follows up https://github.com/doctrine/coding-standard/pull/268 the same way for calls and `use`

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -130,6 +130,7 @@ jobs:
       matrix:
         php-version:
           - "7.2"
+          - "7.3"
```
Comment (Ocramius) [inline .github/workflows/continuous-integration.yml:133]: Do we still need these? They are EOL and abandoned

File: tests/php81-compatibility.patch [hunk 2]
```diff
@@ -374,3 +422,50 @@ index 6e5cca4..1d24bb6 100644
      {
      }
  
+@@ -23,7 +23,7 @@ class TrailingCommaOnFunctions
+     {
+         $var = null;
+ 
+-        $singleLine = static function (int $arg) use ($var): void {
++        $singleLine = static function (int $arg) use ($var,): void {
```
Comment (Ocramius) [inline tests/php81-compatibility.patch:430]: This really doesn't look good :thinking:

File: tests/php81-compatibility.patch [hunk 3]
```diff
@@ -374,3 +422,50 @@ index 6e5cca4..1d24bb6 100644
...
+             var_dump($var);
+         };
+ 
+@@ -37,7 +37,8 @@ class TrailingCommaOnFunctions
+ 
+ $class = new TrailingCommaOnFunctions();
+ 
+-$class->a(1);
++// phpcs:ignore Generic.Functions.FunctionCallArgumentSpacing.NoSpaceAfterComma
++$class->a(1,);
```
Comment (Ocramius) [inline tests/php81-compatibility.patch:440]: Also a bad idea? Unless I'm misunderstanding the purpose of this file :thinking:

**Review by Ocramius — CHANGES_REQUESTED** [review]

File: tests/php81-compatibility.patch (hunk 2 above)
Comment (simPod) [reply tests/php81-compatibility.patch:430]: This is patched input so I can test `,` removal.

```diff
- $singleLine = static function (int $arg) use ($var,): void { // input
+ $singleLine = static function (int $arg) use ($var): void { // result
```

File: tests/php81-compatibility.patch (hunk 3 above)
Comment (simPod) [reply tests/php81-compatibility.patch:440]: also patched input for test so we can test the result is `(1)`

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (simPod) [reply .github/workflows/continuous-integration.yml:133]: Maintenance is kind of burden. I basically have to patch test inputs and outputs for each php version so I can add tests for new sniffs.

**Review by ostrolucky — APPROVED** [review]

**Review by Ocramius — APPROVED** [review]

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (Ocramius) [reply .github/workflows/continuous-integration.yml:133]: Heh, I was suggesting to drop support overall

**Review by derrabus — APPROVED** [review]

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (simPod) [reply .github/workflows/continuous-integration.yml:133]: Yup, that was an agreement from my side :D 

but as discussed https://github.com/doctrine/coding-standard/pull/268#discussion_r901128956

**Pull request merged**